README.md

```diff
@@ -180,6 +180,20 @@ zoxide can be installed in 4 easy steps:
 
    </details>
 
+   <details>
+   <summary>`cmd.exe`</summary>
+
+   > Add this to the **end** of your config file or AutoRun command:
+   >
+   > ```batchfile
+   > zoxide init cmd | cmd /d >nul
+   > ```
+   >
+   > **Note**
+   > zoxide only supports `cmd.exe` with Command Extensions v2 and above.
```

Comment:
It seems Command Extensions is enabled by default on later versions of Windows. Can we remove this from the README, and add a check for it instead in the script? We could throw an informative error here, like:

```
zoxide: requires CommandExtensions v2 to run on cmd.exe, please see the instructions at <URL>
```

Reply:
Sure.